core/src/main/java/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -533,7 +431,16 @@ private String getItemValue(ItemData itemData, CrfVersion crfVersion) {
                 itemValue = itemValue.replaceAll(",", " ");
             }
         }
-        return itemValue;
+        return escapedValue(itemValue);
+    }
+
+    private String escapedValue(String value) {
+        if (value != null) {
```

Comment:
concise way to do this would be
value = value.replaceAll("&", "&amp;").replaceAll(...).replaceAll()

I think it would be more readable and concise.